apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/PatientE2EBase.java

```diff
@@ -831,7 +832,7 @@ public void testPatientByIdentifierHasAccessJsonWithExpectedMdcKeys() throws IOE
             + "|"
             + mbiHash;
 
-    List<String> additionalExpectedMdcKeys = new ArrayList<>(MDC_EXPECTED_BASE_KEYS);
+    List<String> additionalExpectedMdcKeys = new ArrayList<>();
```

Comment:
Did we want to remove all of these? I thought it was just the IncludeIdentifiers one that was problematic.

Reply:
Oh my bad, I thought some of those keys weren't affected.

Reply:
@aschey-forpeople Just HTTP_ACCESS_RESPONSE_DURATION_PER_KB is unaffected in this test and HTTP_ACCESS_RESPONSE_HEADER_CONTENT_LOCATION in another